pipeline-model-definition/src/test/resources/errors/incorrectNestedStagesNPE.groovy

```diff
@@ -0,0 +1,62 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2018, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+pipeline {
+
+    agent any
+
+    stages {
+        stage('PARALLEL-WRAPPER') {
+            parallel {
+                stages {
+                    stage('file-1') {
+                        steps {
+                            sh "touch file-1"
+                        }
+                    }
+                    stage('file-2') {
+                        steps {
+                            sh "touch file-2"
+                        }
+                    }
+                }
+
+                stage('Contains Sequentials') {
+                    stages {
+                        stage('ls') {
+                            steps {
+                                sh "ls"
+                            }
+                        }
+                        stage('ls') {
+                            steps {
+                                sh "ls file-2"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Can you add a newline at the end to avoid Ø?

Reply:
Done!